discovery/openstack/mock_test.go

```diff
@@ -49,15 +51,12 @@ func (m *SDMock) Setup() {
 const tokenID = "cbc36478b0bd8e67e89469c7749d4127"
 
 func testMethod(t *testing.T, r *http.Request, expected string) {
-	if expected != r.Method {
-		t.Errorf("Request method = %v, expected %v", r.Method, expected)
-	}
+	require.Equal(t, expected, r.Method, "Expected request method.")
 }
 
 func testHeader(t *testing.T, r *http.Request, header string, expected string) {
-	if actual := r.Header.Get(header); expected != actual {
-		t.Errorf("Header %s = %s, expected %s", header, actual, expected)
-	}
+	actual := r.Header.Get(header)
+	require.Equal(t, expected, actual, "Expected request header %s.", header)
```

Comment:
This message isn't that clear. We're testing if the header value is expected, not if the header is present, which is what this message makes me think. Maybe something like `Unexpected value for header %s`.